### PR TITLE
feat: Collapsible Panel: Remove title line clamping [GAUD-6356]

### PR DESCRIPTION
**Pull request by margaree**
[Jira ticket](https://desire2learn.atlassian.net/jira/software/c/projects/GAUD/boards/330?selectedIssue=GAUD-6356)

### CODE REVIEW

**Review by dbatiste — APPROVED** [review]

File: components/collapsible-panel/collapsible-panel.js [hunk 1]
```diff
@@ -171,7 +171,6 @@ class CollapsiblePanel extends SkeletonMixin(FocusMixin(RtlMixin(LitElement))) {
 				-webkit-box-orient: vertical;
 				display: -webkit-box;
 				flex: 1;
-				-webkit-line-clamp: 2;
```
Comment (dlockhart) [inline components/collapsible-panel/collapsible-panel.js:174]: There were a [few other changes](https://github.com/BrightspaceUI/core/pull/3299) made when Dave added this... some I think we still need (like `overflow-wrap: anywhere`, but I'm wondering about `overflow-x/y` and a couple others...

Comment (dbatiste) [reply components/collapsible-panel/collapsible-panel.js:174]: Oh yeah, good call. I think `-webkit-box-orient: vertical;` and `display: -webkit-box;` are probably to make the line-clamping work too.

**Review by dlockhart — APPROVED** [review]

File: components/collapsible-panel/collapsible-panel.js (hunk 1 above)
Comment (bearfriend) [reply components/collapsible-panel/collapsible-panel.js:174]: What is the `overflow-wrap: anywhere` getting us here?

Comment (dbatiste) [reply components/collapsible-panel/collapsible-panel.js:174]: Wrapping unbreakable titles.

**Pull request merged**